libbeat/reader/parser/parser.go

```diff
@@ -30,6 +30,7 @@ import (
 	"github.com/elastic/beats/v7/libbeat/reader/multiline"
 	"github.com/elastic/beats/v7/libbeat/reader/readfile"
 	"github.com/elastic/beats/v7/libbeat/reader/readjson"
+	"github.com/elastic/beats/v7/libbeat/reader/syslog"
 )
 
```

Comment:
There should be some documentation for `parsers` where we need to add `syslog`.

Reply:
Where is the documentation for parsers? None of the other parsers seem to have documentation, but I may have missed it. The only references I've seen are in the documentation for the filestream input. (https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-input-filestream.html#_parsers)

Reply:
Ok, that's all I found too. So let's update that list at the least.

Reply:
Can do, I'll also drop in a docs directory and have a file similar to the one for the processor. I'd imagine we'll eventually have a doc section for parsers, so that'll at least help the next person that comes along and works on that.